### PR TITLE
[WIP] 

### PR DESCRIPTION
**Pull request by Nyholm**
* Removed variable not used

### CODE REVIEW

**Comment by Nyholm** [comment]
Do we really need a `BaseBuilder`?

File: src/Command/CacheFlushCommand.php [hunk 1]
```diff
@@ -37,10 +37,10 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        $serviceName = 'aequasi_cache.instance.' . $input->getArgument('instance');
+        $serviceName = sprintf('aequasi_cache.instance.%s.bridge', $input->getArgument('instance'));
```
Comment (aequasi) [inline src/Command/CacheFlushCommand.php]: this is backwards, no?

Comment (Nyholm) [inline src/Command/CacheFlushCommand.php]: I want the bridge here, because I want to use the `clear`. 
Backwards, how?

Comment (aequasi) [inline src/Command/CacheFlushCommand.php]: the bridge uses `flushAll`, the non-bridge uses `clear`

**Comment by aequasi** [comment]
Not really? But it makes the other builders cleaner?

**Comment by Nyholm** [comment]
It made sens when we had multiple builders... Anyhow.. we can leave it

**Comment by Nyholm** [comment]
I want to remove the class parameters in the service file. Is that Okey?

File: src/DependencyInjection/Compiler/RouterCompilerPass.php [hunk 2]
```diff
@@ -32,6 +32,6 @@ protected function prepare()
 
         $def = $this->container->findDefinition('router');
         $def->setClass('Aequasi\Bundle\CacheBundle\Routing\Router');
-        $def->addMethodCall('setCache', [new Reference('aequasi_cache.instance.' . $instance)]);
+        $def->addMethodCall('setCache', [new Reference(sprintf('aequasi_cache.instance.%s.bridge', $instance))]);
```
Comment (aequasi) [inline src/DependencyInjection/Compiler/RouterCompilerPass.php]: The bridge is not PSR-6 CacheItemPoolInterface

**Comment by aequasi** [comment]
Uh, maybe? Try it

**Pull request merged**